packages/dd-trace/src/exporters/common/docker.js

```diff
@@ -10,43 +10,25 @@ const uuidSource =
 '[0-9a-f]{8}[-_][0-9a-f]{4}[-_][0-9a-f]{4}[-_][0-9a-f]{4}[-_][0-9a-f]{12}|[0-9a-f]{8}(?:-[0-9a-f]{4}){4}$'
 const containerSource = '[0-9a-f]{64}'
 const taskSource = '[0-9a-f]{32}-\\d+'
-const lineReg = /^(\d+):([^:]*):(.+)$/
+const lineReg = /^(\d+):([^:]*):(.+)$/m
 const entityReg = new RegExp(`.*(${uuidSource}|${containerSource}|${taskSource})(?:\\.scope)?$`, 'm')
 
-const cgroup = readControlGroup()
-const entityId = getEntityId()
-const inode = getInode()
+let inode = 0
```

Comment:
Why this unrelated change? In the PR description you mentioned that you think this is more readable, but I'm not sure I understand why. I definitely prefer being able to be able to quickly understand where a value comes from by isolating its provenance to a function call and knowing it won't be changed afterwards thanks to `const` instead of having to read through the whole file to find any code that could be changing the value.

Reply:
The former code was more than twice as long and abstracting it in methods meant I had to check if these were called more than once and from where they were called. Now it's just sequential simple code :)